api/wft-complete-metadata.md

```diff
@@ -0,0 +1,133 @@
+# Workflow Task Completion Metadata
+
+We have a need to attach various bits of metadata to workflow task completions. There are a number
+of usecases in which the SDK would like to save some information for itself, or tell server about
+something, but this information does not influence replay or the reconstitution of workfow state in
```

Comment:
For SDK internal versioning won't this feature effect replay/reconstitution of workfow state? Might be clearer to say user workfow state?

Reply:
Yeah, which is what I get at later on at the bottom.